isodatetime/data.py

```diff
@@ -1048,10 +1044,10 @@ def apply_time_zone_offset(self, offset):
                     offset.minutes / float(CALENDAR.MINUTES_IN_HOUR))
             else:
                 self.minute_of_hour += offset.minutes
-            self._tick_over()
+            self.tick_over()
```

Comment:
In `TimePoint.add_truncated`, the method is invoked as `new.tick_over`, which is a bit bad according to pylint.